api/v1/helpers/nouns/users.js

```diff
@@ -23,6 +23,9 @@ module.exports = {
     POST: `Create a new ${m}`,
     PUT: `Overwrite all attributes of this ${m}`,
   },
+  scopeMap: {
+    withoutSensitiveInfo: 'withoutSensitiveInfo',
```

Comment:
I feel like it would be better to have default scope exclude the password field, and only include the password in the response when it is explicitly needed for authentication (we have a separate user story for this)